azure-pipelines.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       options: --privileged
 
     steps:
+    # todo(alessandro): move this inside the docker image
```

Comment:
Rather than having `todo`s throughout the code base, any chance this could be moved to a GH issue and assigned to the relevant party (@alessandrogario ;))

Reply:
The todo is only temporary and I'll remove it when I remove the draft state from the pr! :smile: